vlingo-e2e-sys-airline-ops/frontend/src/routes/fleet-crew.svelte

```diff
@@ -0,0 +1,119 @@
+<svelte:head>
+	<title>Fleet Crew</title>
+</svelte:head>
+
+<script>
+	import { Select, Button, Dialog, Row, Alert } from 'svelte-materialify/src';
+	import CardForm from '../components/CardForm.svelte';
+	import { Api } from "../api";
+	import { flights } from "../stores/flights.js";
+	import { inventories } from "../stores/inventory.js";
+	import { fleetcrews } from "../stores/fleetcrew.js";
+	import { required } from "../util/validators.js";
+
+	let isDialogActive = false;
+	let valid = false;
+
+	let selectedAircraft = "";
+	let selectedFlight = "";
+
+	let formData = {
+		aircraftId: "05e5b41c-1fc7-4946-b04a-fb7a43d9d119",
+		carrier: "EMB",
+		flightNumber: "1983",
+		tailNumber: "2011"
+	}
+
+	const submit= async () => {
+		const res = await Api.post("/fleetcrew/aircrafts/", formData);
+		console.log(res);
+		if (res) {
+			$fleetcrews = [...$fleetcrews, res]
+			toggleDialog()
+		}
+	}
+
+	const toggleDialog = () => {
+		isDialogActive = !isDialogActive;
+	}
+
+	$: aircrafts = $inventories.map((inv => {
+		return {
+			name: inv.manufacturerSpecification.manufacturer,
+			value: inv.id,
+			...inv
+		};
+	}));
+
+	$: flightsForSelect = $flights.map((f => {
+		return {
+			name: f.id,
+			value: f.id,
+			...f
+		};
+	}));
+
+	$: {
+		if (selectedAircraft) {
```

Comment:
I actually don't know, is there a difference or can you just do:
`$: if (selectedAircraft) { ... }` ?